app/index.js

```diff
@@ -1447,6 +1454,18 @@ module.exports = JhipsterGenerator.extend({
             this.removefile('profile_prod.gradle');
             this.removefile('sonar.gradle');
             this.removefile('yeoman.gradle');
+        },
+
+        writeEntitiesFiles: function () {
+            if (this.skipEntities) {
+              return;
+            }
+            var entities = this.config.get('entities');
+            if (entities !== undefined) {
```

Comment:
I had started by doing forEach but I had an issue. It was probably unrelated, so I will check again